### PR TITLE
Skip `remote-view` on CFX

### PR DESCRIPTION
**Pull request by chrisnevers**

### CODE REVIEW

File: examples/remote-view/index.mjs [hunk 1]
```diff
@@ -7,7 +7,7 @@ import * as fs from 'fs';
 const stdlib = loadStdlib(process.env);
 
 (async () => {
-  if ( stdlib.connector === 'ALGO' ) { return; }
+  if ( ['ALGO', 'CFX'].includes(stdlib.connector) ) { return; }
```
Comment (jeapostrophe) [inline examples/remote-view/index.mjs:10]: I think my preference is to continue running it on CFX, but add it to xfail, so we don't forget about the error by losing it in some old Jira card in the backlog.

File: examples/remote-view/index.mjs [hunk 2]
```diff
@@ -37,8 +42,9 @@ const stdlib = loadStdlib(process.env);
       keepGoing: async () => {
         const kg = count < 5;
         const ctcAddr = await ctcAlice.getContractAddress();
-        const x = await contract.callStatic["callView"](ctcAddr);
-        console.log(`Compare: `, x.toNumber(), count);
+        const f = (stdlib.connector == 'CFX') ? contract : contract.callStatic;
```
Comment (jeapostrophe) [inline examples/remote-view/index.mjs:45]: Perhaps this means that we should change `cfxers` to support `callStatic`?

Comment (chrisnevers) [reply examples/remote-view/index.mjs:45]: I'll make a ticket for that

File: examples/remote-view/index.mjs (hunk 1 above)
Comment (chrisnevers) [reply examples/remote-view/index.mjs:10]: Looks like it was added to xfail two days ago.. I'll close this PR

**Pull request closed without merging**